origami.json

```diff
@@ -3,7 +3,7 @@
 	"origamiType": "component",
 	"supportStatus": "active",
 	"brands" : [
-		"master"
+		"main"
```

Comment:
This isn't a branch name unfortunately, it is a brand name -- the brand name "master" is being renamed at some point but that work hasn't been finished just yet

```suggestion
		"master"
```

Reply:
so renaming this will fall under your migration work in future?

Reply:
Yes 👍